src/rpdk/core/contract/suite/handler_list.py

```diff
@@ -0,0 +1,20 @@
+# fixture and parameter have the same name
+# pylint: disable=redefined-outer-name
+
+import pytest
+
+# WARNING: contract tests should use fully qualified imports to avoid issues
+# when being loaded by pytest
+from rpdk.core.contract.interface import Action, OperationStatus
+
+
+@pytest.mark.list
+def contract_list_empty(resource_client):
```

Comment:
can this test fail at the moment? either there arent resources and we "assert True" or we skip it? might be good to just  remove completely